registry.tcl

```diff
@@ -762,4 +762,9 @@ extension EGL_EXT_surface_compression {
     flags       public
     filename    extensions/EXT/EGL_EXT_surface_compression.txt
 }
+extension EGL_EXT_display_alloc {
+    number      148
```

Comment:
This is going to end up as extension 149 (if it's the next one that is merged). Actually I guess egl.xml / registry.tcl do need to be rebased - I had forgotten that adding to the end of the file always generates conflicts between two branches.